icons/test-tube.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M21,6L5.4,21.4c-0.8,0.8-2,0.8-2.8,0c-0.8-0.8-0.8-2,0-2.8L18,3" />
+  <path d="M17.1,2L22,6.9" />
+  <line x1="5.5" y1="16" x2="10.5" y2="16" />
```

Comment:
This is a very basic fix for pixel perfection keeping pretty much the same design:
```suggestion
  <path d="M21 6 5.62 21.38c-.83.83-2.17.83-3 0v0a2.12 2.12 0 0 1 0-3L18 3" />
  <path d="m17 2 5 5" />
  <path d="M15 12H9" />
```

Although I think maybe we could fatten it up by a single unit to make the roundness more pronounced on 100% (and potentially also add a matching pair of vertical test tubes as an alternate):
![image](https://user-images.githubusercontent.com/17746067/228445419-fb6cb68b-047d-4978-a086-d656031622fd.png)

Reply:
Yeah, like that idea…